components/suggestion/switch_channel_provider.jsx

```diff
@@ -218,16 +212,40 @@ function quickSwitchSorter(wrappedA, wrappedB) {
 
     const aStartsWith = aDisplayName.startsWith(prefix);
     const bStartsWith = bDisplayName.startsWith(prefix);
-    if ((aStartsWith && bStartsWith) || (!aStartsWith && !bStartsWith)) {
-        //
-        // MM-12677 When this is migrated this needs to be fixed to pull the user's locale
-        //
+
+    // Open channels user havent interacted should be at the  bottom of the list
```

Comment:
Do we need an explicit case for this? Shouldn't those be automatically sent to the bottom of the list based on the other conditions?

Reply:
Ya, it is to explicitly send open channels even if they match to the bottom of the list hence at the top of the conditions